docs/ref/csrf.txt

```diff
@@ -211,6 +211,8 @@ The error page, however, is not very friendly, so you may want to provide your
 own view for handling this condition.  To do this, simply set the
 :setting:`CSRF_FAILURE_VIEW` setting.
 
+CSRF failures are logged to ``django.requests`` as warnings.
```

Comment:
Use: 
```
:ref:`django-request-logger`
```
(this will render correctly as django.request instead of the nonexistent django.requests)

Even if we decide to change to use the security logger, I guess we could accept this addition for older versions of the docs.